web/templates/pages/members.html

```diff
@@ -20,13 +19,40 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
-					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
+					
+					<li class="mentor-cat">
+					<a href="{{ member.githubuser }}">
```

Comment:
@anuschka Continuing from previous comments for the same issue ... This link still doesn't work (and won't be merged until it does). We're saving GitHub usernames, not GitHub profile URLs in user profiles. I'm *quite* sure you can figure out a way to compose the right URL from the username.

Also, we're using GitHub login, so technically you can't login with a username that doesn't exist ...